CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.22)
 project(rdf4cpp VERSION 0.0.49)
-set(POBR_VERSION 2)  # Persisted Object Binary Representation
+set(POBR_VERSION 3)  # Persisted Object Binary Representation
```

Comment:
Don't need a POBR version bump, the change in BiDirFlatMap does nothing on relevant architectures

Reply:
I'll revert it